server/status.go

```diff
@@ -84,7 +87,18 @@ func (s *statusServer) RegisterHandlers(mux *http.ServeMux) {
 	mux.HandleFunc(statusTransactionsKeyPrefix, s.handleTransactionStatus)
 }
 
-// TODO(shawn) lots of implementing - setting up a skeleton for hack week.
+// marshalJSON marshals the provided obj into JSON format. If the
+// HTTP request parameter has "indent" set, using MarshalIndent.
```

Comment:
I would make indentation the default; the performance cost rarely matters and it's better to program any robot that will be fetching this page frequently to disable indentation that to make humans remember to turn it on.